Lib/importlib/_bootstrap.py

```diff
@@ -51,18 +51,132 @@ def _new_module(name):
 
 # Module-level locking ########################################################
 
-# A dict mapping module names to weakrefs of _ModuleLock instances
-# Dictionary protected by the global import lock
+# For a list that can have a weakref to it.
+class List(list):
+    pass
+
+
+# From weakref.py with some simplifications.
```

Comment:
You can remove these methods to make the code shorter: ``__getitem__()``, ``__setitem__()``, ``__delitem()``, ``__repr__()``. You may also remove comments to make the code shorter.